app/controllers/MainController.scala

```diff
@@ -49,8 +49,8 @@ class MainController @Inject() (actorSystem: ActorSystem,
     val dslinkStats = getDSLinkCounts
     val uplinkStats = getUplinkCounts
     for {
-      down <- dslinkStats
-      up <- uplinkStats
+      down <- dslinkStats.recover{case e => DSLinkStats(Map.empty)}
+      up <- uplinkStats.recover{case e => DSLinkStats(Map.empty)}
```

Comment:
perhaps `NonFatal(e)` instead of just `e`

Reply:
Thanks, fixed.